jsonld-addon-integration/pom.xml

```diff
@@ -20,17 +23,81 @@
     <compaction.preload.file.farm>${project.basedir}/src/test/resources/context.jsonld</compaction.preload.file.farm>
   </properties>
 
-  <profiles>
-    <profile>
-      <id>standard</id>
-      <properties>
-        <fcrepo.dynamic.test.port>8080</fcrepo.dynamic.test.port>
-        <fcrepo.dynamic.jms.port>61616</fcrepo.dynamic.jms.port>
-        <fcrepo.dynamic.stomp.port>61613</fcrepo.dynamic.stomp.port>
-      </properties>
-    </profile>
-  </profiles>
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.pass</groupId>
+      <artifactId>jsonld-addon-filters</artifactId>
+      <version>${project.version}</version>
+      <classifier>shaded</classifier>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.fcrepo</groupId>
+      <artifactId>fcrepo-webapp</artifactId>
+      <version>${fcrepo.version}</version>
+      <type>war</type>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.jena</groupId>
+      <artifactId>jena-fuseki-war</artifactId>
+      <version>2.4.1</version>
```

Comment:
Not a big deal, but is there a reason why this dependency has its version hardcoded instead of using a property?

Reply:
we can fix it at some point, i am sure there are others in other projects too. i think it's outside the scope of this ticket